dotnet-8.0.404.yaml

```diff
@@ -0,0 +1,99 @@
+package:
+  name: dotnet-8.0.404
+  version: 8.0.404
+  epoch: 1
+  description: ".NET SDK"
+  copyright:
+    - license: MIT
+  resources:
+    cpu: 2
```

Comment:
do we know that dotnet needs these resources to work at all?

Reply:
it downloads lots of stuff; and links lots of stuff; these resources were set by infra people before based on past builds.